rancher/v1.4/en/cattle/secrets/index.md

```diff
@@ -5,75 +5,95 @@ version: v1.4
 ```
-docker run -d --restart unless-stopped -p 8080:8080 -e VAULT_ADDR=https://<vault_server> -e VAULT_TOKEN=<token_for_vault_access> rancher/server
-```
 
-Once the server is up and running, on the Admin-Settings tab under `Advanced` change the value of `secrets.backend` from `localkey` to `vault`
-> Note: as of this time you can not switch between backend. 
-
-> Note: The token used for the Vault backend should be scoped with a policy to just the Rancher servers use of Vault Transit endpoints for:
-> 
->  transit/random/_key_
-> 
->  transit/hmac/_key_
-> 
->  transit/encrypt/_key_
-> 
->  transit/decrypt/_key_
-> 
->  transit/verify/_key_
-> 
-> Where _key_ will need to be `rancher` at this time.
-
-In order to consume secrets in containers, you need to launch the `Rancher Secrets` experimental catalog entry in the 'Library' catalog. Otherwise, you can create secrets, but you can't consume them.
+> **Note:** Verify that you are running the desired [Rancher server tag]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/installing-rancher/installing-server/#rancher-server-tags).
+
+Once Rancher server is up, you will need to configure a setting within Rancher. Under **Admin** -> **Settings** -> **Advanced**, update the `secrets.backend` value. By default, it will have `localkey` and it needs to be changed to `vault`. 
+
+> **Note:** Currently, Rancher does not support switching between encryption backend types. 
 
 ### Creating Secrets
 
-Secrets are created and scoped at an environment level. Applications can share these secrets, for instance a database password `db_password` can be used in both the database container and the Wordpress container. 
+Secrets are created and scoped at an environment level. Any containers can share the same secrets. For example, a database password, i.e. `db_password`, can be used in a database container and a Wordpress container. 
 
-To create a secret from the UI go to the Infrastructure->Secrets tab. There you can create a key value pair for the secret. 
+Currently, secrets can only be created in the UI, but Rancher is looking to add in CLI support. 
 
-Secrets can not be edited or updated. In order to update a secret the user has to create a new, or delete/recreate the password and redeploy the services with the updated secret.
+Go to **Infrastructure** -> **Secrets**. Create a key value pair for a secret. 
 
-Deleting the secret in Rancher does not remove the secret or file from the host.
+Once a secret is created, it **cannot** be edited or updated. If you need to change an existing secret, the only way to do so is to delete the secret. You can create either a brand new key value pair or use the same key with a new value. After the secret has been updated, any services using the secret will need to be re-launched with the updated secret. 
+
+### Deleting Secrets
+
+Secrets can be deleted from Rancher, but it does remove the secret (i.e. file) from the _??host (or container or both)??_.
```

Comment:
Both. Neither the container or host volume. When the container is deleted the volume and contents are removed from the host.